specs/progress.md

```diff
@@ -0,0 +1,142 @@
+# fast-progress
```

Comment:
I know this was done on switch, but I think we need to standardize this to just "Progress", fast-progress is the tag name.